src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-details/iscsi-target-details.component.spec.ts

```diff
@@ -97,15 +97,18 @@ describe('IscsiTargetDetailsComponent', () => {
         {
           children: [{ id: 'disk_rbd_disk_1', value: 'rbd/disk_1' }],
           settings: {
-            cssClasses: { expanded: 'fa fa-fw fa-hdd-o fa-lg', leaf: 'fa fa-fw fa-hdd-o' },
+            cssClasses: { expanded: 'fa fa-fw fa fa-lg fa fa-hdd-o', leaf: 'fa fa-fw fa fa-hdd-o' },
```

Comment:
What about using this?
```suggestion
            cssClasses: { expanded: _.join([Icons.width, Icons.large, Icons.disk], ' '), leaf: _.join([Icons.width, Icons.disk], ' ') },
```